src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -10,7 +10,7 @@ namespace Internal.Runtime.CompilerHelpers
     /// <summary>
     /// These methods are used to throw exceptions from generated code.
     /// </summary>
-    internal static class InteropHelpers
+    public static class InteropHelpers
```

Comment:
Well, there is still the `InternalsVisibleToAttribute` that will mess with the assumption that "things that are not public can't be accessed from outside of the module".